azurerm/data_source_app_configuration_value.go

```diff
@@ -0,0 +1,98 @@
+package azurerm
+
+import (
+	"fmt"
+	"time"
+
+	appconf "github.com/Azure/azure-sdk-for-go/services/appconfiguration/mgmt/2019-10-01/appconfiguration"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tags"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmAppConfigurationValue() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmAppConfigurationValueRead,
+
+		Timeouts: &schema.ResourceTimeout{
+			Read: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"app_conf_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: azure.ValidateResourceID,
+			},
+
+			"key": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"label": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"value": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"content_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"last_modified": {
```

Comment:
I normally like to steer away from time based values being held in Terraform since they tend to be more clutter rather than a benefit. Is there a specific use case you envision this being used in? If not, would you be ok with removing it?

Reply:
Agree :+1: